Added Documentation for IBM Quantum API

iglesias-cardinale opened the pull request:
I added an actual markdown file for what I put in issue #26. This should be useful for anyone who is having trouble connecting their code to the IBM Quantum Computer.

laserlab commented on IBM Quantum API Information.md:
```diff
@@ -0,0 +1,31 @@
+
+This can be done by including the line 
+
+"**QiskitRuntimeService.save_account(token="YOUR_API_TOKEN", channel="ibm_quantum")** 
```
Prettier markdown would be to use a code markup such as:
```markdown

> ```python
> QiskitRuntimeService.save_account(token="YOUR_API_TOKEN", channel="ibm_quantum")
> ```

```

laserlab commented on .devcontainer/qiskit/devcontainer.json:
Looks like this should not be part of the PR. Please remove

laserlab reviewed (changes requested):
- remove devcontainer.json
 - better markdown code formatting

laserlab commented:
add label and milestone to PR

iglesias-cardinale committed:
Create IBM Quantum API Information.md

Documentation on how to connect your code in python (Qiskit) to IBM's Quantum computer using an API with proper security

laserlab reviewed (approved):
looks good now

iglesias-cardinale commented:
I'm glad this worked out. I basically made an entirely new branch in my forked repository and then tried to merge which I think was unnecessary.
